### DISCUSSION TITLE
Welcome to fluvio Discussions!

### DISCUSSION THREAD

**Discussion by sehz** [fluvio-community/fluvio#1009] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicholastmosher** [comment]
Hi everyone 👋 my name's Nick (not to be confused with @nacardin, who is also Nick), and I work at InfinyOn on Fluvio. I spend most of my time on the open-source core Fluvio project: I've done a lot of work on the CLI, the Rust client crate, the Fluvio installer (cluster) crate, and have started working a little bit in the cluster components as well, the SC and SPU.

One of my favorite parts of this project is API design, trying to figure out what the needs of the users are and designing an interface that intuitively meets those needs while also minimizing the opportunity for errors. If you find yourself struggling to accomplish your use-cases, feel free to open an issue and tag me in it, I'll be happy to work with you to brainstorm solutions!

We're looking forward to meeting you all!
